lib/toolbox/Taskfile.yaml

```diff
@@ -1,59 +1,96 @@
 # See more information in docs/toolbox.md
 version: "3"
 vars:
-  IMAGE_NAME: toolbox
-  LATEST_IMAGE_TAG: "{{.IMAGE_NAME}}:latest"
-  VERSION_IMAGE_TAG: "{{.IMAGE_NAME}}:{{.VERSION}}"
   VERSION:
     sh: git rev-parse --short HEAD
-  REGISTRY_URL_FULL: "{{.REGISTRY_URL}}/{{.LATEST_IMAGE_TAG}}"
-  CONTAINERFILE_PATH: '{{.CONTAINERFILE_PATH | default "./Dockerfile"}}'
+  TOOLBOX_NAME: '{{.TOOLBOX_NAME | default "toolbox"}}'
+  LATEST_IMAGE_TAG: "{{.TOOLBOX_NAME}}:latest"
+  VERSION_IMAGE_TAG: "{{.TOOLBOX_NAME}}:{{.VERSION}}"
+  REGISTRY_LATEST_URL: "{{.REGISTRY_URL_BASE}}/{{.LATEST_IMAGE_TAG}}"
+  REGISTRY_VERSION_URL: "{{.REGISTRY_URL_BASE}}/{{.VERSION_IMAGE_TAG}}"
+  CONTAINERFILE_PATH: '{{.CONTAINERFILE_PATH | default "./Containerfile"}}'
   AWS_REGION: '{{.AWS_REGION | default "us-east-1"}}'
 
 tasks:
-  all:
+  auth:
+    desc: Authenticate to AWS ECR
+    internal: true
+    requires:
+      vars:
+        - AWS_REGION
+        - REGISTRY_URL_BASE
+    cmds:
+      - aws ecr get-login-password --region {{.AWS_REGION}} | docker login --username AWS --password-stdin {{.REGISTRY_URL_BASE}}
+
+  install:
+    desc: Install wrapper script from geodesic container
+    internal: true
+    requires:
+      vars:
+        - TOOLBOX_NAME
+    cmds:
+      - docker run --rm {{.REGISTRY_LATEST_URL}} init | bash -s {{.TOOLBOX_NAME}}
```

Comment:
it's impossible to understand what this task does from the code here. Please add a longer description, or if you can, pull the install wrapper script outside of the container to somewhere that it can be viewed

Reply:
Great point -- I'll make this clearer via `desc` + `summary`. Thanks for the callout.